python/pyspark/streaming/util.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, ctx, func, *deserializers):
         self.ctx = ctx
         self.func = func
         self.deserializers = deserializers
+        self._rdd_wrapper = lambda jrdd, ctx, ser: RDD(jrdd, ctx, ser)
+
+    def rdd_wrapper(self, func):
```

Comment:
Shouldnt this be called `set_rdd_wrapper`? Can you put some comments on what it does and why it may require setting a custom wrapper?